share/spice/dota2/dota2_hero.handlebars

```diff
@@ -0,0 +1,71 @@
+<div class="gw">
+    <div class="g third hero-image">
+        <img class="whole--screen-s" src="{{image}}" title="{{name}}" />
```

Comment:
I think we'll need to shrink the height of the image a bit here, right now the whole IA doesn't fit within the first page.

Alternatively we need to use our content chomping classes/JS to limit the height of the initial result and allow the user to expand it. If they want to see more info